src/dbnode/storage/index/block.go

```diff
@@ -234,7 +234,6 @@ func NewBlock(
 		iopts,
 	)
 
-	aggAdded := opts.InstrumentOptions().MetricsScope().Counter("aggregate-added-counter")
```

Comment:
why remove this?

Reply:
This metric is essentially a copy of the total aggregate results metric [here](https://github.com/m3db/m3/pull/3137/files#diff-ddb6541a09168df2ec556d66c890cbe9032c61b8d5ffcd13f1db6a49bf952e42R58), so not too useful on its own